### PR TITLE
React native app "focus"

### PR DESCRIPTION
**Pull request by gabriel**
Support app focus from react-native.

(Renamed window:changedFocus action to app:changedFocus.)

Fixes https://keybase.atlassian.net/browse/DESKTOP-3143 which allows conversations to be marked as read when it is selected and comes in focus, same as desktop app.

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

File: shared/constants/app.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// @flow
+import type {NoErrorTypedAction} from './types/flux'
+
+export type ChangedFocus = NoErrorTypedAction<'app:changedFocus', boolean>
```
Comment (chrisnojima) [inline shared/constants/app.js]: lets always have a payload object and not a naked value
ie
{focused: boolean}

**Comment by gabriel** [comment]
Made payload boolean arg `focused`.

**Comment by chrisnojima** [comment]
we should try and merge master sep. from our code changes. hard to tell whats yours and others in that commit

**Pull request merged**